airflow/providers/common/sql/operators/sql.py

```diff
@@ -112,6 +124,61 @@ def get_db_hook(self) -> DbApiHook:
         return self._hook
 
 
+class SQLExecuteQueryOperator(BaseSQLOperator):
```

Comment:
Please create unit tests for this operator. You should not rely on unit tests made for all the different DB operators because at some point we are going to remove them (since they are now deprecated). Along the operators we'll remove, we'll remove as well the unit tests associated to them